buildbot/master/files/config/master.cfg

```diff
@@ -170,7 +170,8 @@ c['builders'] = [
     DynamicServoBuilder("arm64", CROSS_SLAVES, envs.build_arm64),
     DynamicServoBuilder("linux-dev", LINUX_SLAVES, envs.build_linux),
     DynamicServoBuilder("linux-nightly", LINUX_SLAVES, envs.build_linux),
-    DynamicServoBuilder("linux-rel", LINUX_SLAVES, envs.build_linux),
+    DynamicServoBuilder("linux-rel", LINUX_SLAVES
```

Comment:
Pffft... Thanks, done, should learn to be more careful when pushing fixups, sigh.

Are there docs in how to install the required deps and test changes to this file, btw?